Fix for the prior posterior plot incorrectly switching the bf10 and bf01 label

JTPetter opened the pull request:
Fixes jasp-stats/jasp-issues#796

https://github.com/jasp-stats/jaspSummaryStatistics/pull/3

JTPetter committed:
Fix for the prior posterior plot incorrectly switching the bf10 and bf01 label

Fixes jasp-stats/jasp-issues#796

JTPetter committed:
Remove unintended style changes

akashrajkn commented on JASP-Engine/JASP/R/commonsummarystatsttestbayesian.R:
```diff
@@ -361,7 +361,7 @@
     paired                    = FALSE,
     BF10user                  = BFlist[["BF10"]],
-    yAxisLegendRobustnessPlot = BFPlots,
+    yAxisLegendRobustnessPlot = bfPlotsH1H0,
```
This causes the unit tests to fail. (The robustness plot expects a string. Can you make the necessary change?)

JTPetter committed:
Fix for the prior posterior plot incorrectly switching the bf10 and bf01 label

Fix for the prior posterior plot incorrectly switching the bf10 and bf01 label and new unit test

AlexanderLyNL commented:
@JTPetter could you please also make this PR to the module?
